src/ReportManager/DetailsSection/SchemaForm/fields/Text/index.test.js

```diff
@@ -3,7 +3,7 @@ import userEvent from '@testing-library/user-event';
 
 import { FORM_ELEMENT_TYPES, ROOT_CANVAS_ID, TEXT_ELEMENT_INPUT_TYPES } from '../../constants';
 import { render, screen } from '../../../../../test-utils';
-import { SchemaFormContext } from '../../SchemaFormContext';
+import SchemaFormContext from '../../SchemaFormContext';
```

Comment:
**Related to AJV validations**
Tests updated with the addition of the error state and the aria tags.